docs/antora/modules/ROOT/pages/management/docker.adoc

```diff
@@ -204,10 +217,126 @@ If you wish to take a snapshot of your Docker installation, the details can be f
 
 == Logging
 
-To check the logs for the different microservices of Lisk Service, use the command `docker container logs CONTAINER`, where `CONTAINER` is the respective Docker container that holds the logs you wish to view.
+The possibility exists here to either use the `docker` commands or the `make` commands to check the logs, and both options are listed here below.
+
+For `docker`, to check the logs for the different microservices of Lisk Service, use the command `docker container logs CONTAINER`, where `CONTAINER` is the respective Docker container that holds the logs you wish to view.
 
 For example, to see the logs for the Gateway microservice, execute the following command:
 
 ----
 docker container logs lisk-service_gateway_1
 ----
+
+To check the logs for different microservices using the `make` commands, the following commands listed below can be used:
+
+.Displays the logs for all the microservices that have been logged so far, and that are currently existing
+----
+make logs
+----
+
+.Displays the latest logs for all the microservices
+----
+make logs-live
+----
+
+.Displays the logs for the specified microservice.
+----
+make logs-blockchain-connector
+----
+In the above example, it will display logs only for the `blockchain-connector` microservice.
+
+.Displays the latest logs for the specified microservice.
+----
+make logs-live-blockchain-connector
+----
+In the above example, it will display latest logs only for the `blockchain-connector` microservice.
+
+
+Replace the `blockchain-connector` with the specific service name required.
+
+
+== Upgrade Lisk Service
+
+To upgrade the Docker container to a desired version, please follow one of the two options below. To find all the tagged versions for Lisk Service, please check the available tags (https://github.com/LiskHQ/lisk-service/tags) on GitHub.
+
+==== Option A - Download pre-built images from DockerHub
```

Comment:
Option A should be the recommended way to upgrade Lisk Service, imho Option A and B should be switched, because B looks easier

Reply:
Also, imho it would be nice to use tabs to switch between Option A & B

Reply:
OK, I have now added tabs.

Reply:
OK